app/javascript/packs/application.js

```diff
@@ -3,7 +3,9 @@ import main from '../mastodon/main';
 if (!window.Intl || !Object.assign || !Number.isNaN ||
     !window.Symbol || !Array.prototype.includes) {
   // load polyfills dynamically
-  import('../mastodon/polyfills').then(main);
+  import('../mastodon/polyfills').then(main).catch(e => {
+    // console.log(e); // no-op
+  });
```

Comment:
In other words, if this had been there when I was debugging, it would have helped me out.